packages/fints/src/segments/hkcdb.ts

```diff
@@ -0,0 +1,35 @@
+import { Format } from "../format";
+import { SegmentClass } from "./segment";
+import { SEPAAccount } from "../types";
+
+export class HKCDBProps {
+    public segNo: number;
+    public version: number;
+    public account: SEPAAccount;
+    public painFormats: string[];
+    public touchdown: string;
+}
+
+/**
+ * HKCDB (SEPA-Dauerauftragsbestand abrufen)
+ * Section C.10.2.3.4
+ */
+export class HKCDB extends SegmentClass(HKCDBProps) {
+    public type = "HKCDB";
+
+    protected serialize() {
+        const { account, touchdown, painFormats } = this;
+        const { iban, bic } = account;
+        const pain0101: string = painFormats.find(x => x.startsWith("urn") && x.indexOf("pain.001.001.03") !== -1);
+        const pain0103: string = painFormats.find(x => x.startsWith("urn") && x.indexOf("pain.001.003.03") !== -1);
+        return [
```

Comment:
Please throw an error here if no pain message was found.

Reply:
Yes, there should be an error in that case. Any tip how I can mock this "painFormats" array in the `test-pin-tan-client.ts`? I tried with this line, but it doesn't work...
```
jest.spyOn(Response.prototype, "painFormats", "get").mockReturnValue(["pain.001.003.03"]);
```